var/spack/repos/builtin/packages/py-markov-clustering/package.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyMarkovClustering(PythonPackage):
+    """Implementation of the Markov clustering (MCL) algorithm in python"""
+
+    homepage = "https://github.com/GuyAllard/markov_clustering"
+    pypi = "markov_clustering/markov_clustering-0.0.6.dev0.tar.gz"
+
+    license("MIT", checked_by="A-N-Other")
+
+    version(
+        "0.0.6.dev0", sha256="8f72eee0ee5d9bfbab1b28bbfa95eaa020b2bba64b528ce45030b8b4300ecf33"
+    )
+
+    depends_on("py-setuptools", type="build")
+
+    depends_on("py-numpy", type=("build", "run"))
+    depends_on("py-scipy@0.19.0:", type=("build", "run"))
+    depends_on("py-scikit-learn", type=("build", "run"))
+    depends_on("py-networkx", type=("build", "run"))
+    depends_on("py-matplotlib", type=("build", "run"))
```

Comment:
These are optional, I would either remove them or add a `+drawing` variant that defaults to false to match the behavior of `pip install markov-clustering`

Reply:
Done.